pennylane/operation.py

```diff
@@ -774,7 +857,7 @@ class Tensor(Observable):
 
     def __init__(self, *args):  # pylint: disable=super-init-not-called
 
-        self._eigvals = None
+        self._eigvals_cache = None
```

Comment:
This is just so it doesn't clash with the inherited `self._eigvals` class method?

Reply:
Could be nice to add to the docstring as a side note that the `_eigvals` class method is not implemented here (since it depends on `_matrix` which can also be added). This is simply due to inheritance, but a note can serve as a convenient reminder that for `Tensor` these two methods would not make sense.